### DISCUSSION TITLE
Padloc does not load

### DISCUSSION THREAD

**Discussion by AlexTheKingYT14** [padloc/padloc#625] [Q&A] [answered]
Hello, someone helped me to install Padloc on a DigitalOceans server, but I have a problem, as soon as I go to the site where Padloc is hosted, I have an infinite load:

<img width="1280" alt="Capture d’écran 2022-11-13 à 14 26 31" src="https://user-images.githubusercontent.com/59262297/201524088-8f844b22-e94d-41aa-bd97-fb83cb3d82be.png">

Can anyone help me please? There are no errors in the console...

**Comment by MaKleSoft** [comment]
> There are no errors in the console...

Hm, that seems unlikely. Can you double-check, please? Also, can you check the network tab?

**Reply by AlexTheKingYT14** [reply]
I double-check and there is nothing, and in the network tab of my Droplet, there is no firewall, if you want to check, add me on Discord, so I can give you the password (AlexTheKing#0001).
I don't know what do to actually...

**Reply by BrunoBernardino** [reply]
@AlexTheKingYT14 what browser are you using, and is there a public URL for us to look at? Showing a screenshot of the browser's console might help.

**Reply by AlexTheKingYT14** [reply]
I'm using Safari on Mac, the URL of the website is: http://pass.animacraft.fr
I don't know how to show you the console, because it was another person who installed padloc for me and I don't know how he did it

**Reply by BrunoBernardino** [reply]
The console is your own browser's console @AlexTheKingYT14 . This is what I can see, for example:
![Screenshot from 2022-11-13 18-56-17](https://user-images.githubusercontent.com/1239616/201539169-10412da7-83ec-498c-99cd-bd8bb3fb66a0.png)

That website was compiled for "localhost", which is why the CSP (`Content-Security-Policy`) is failing and not allowing it to load. It needs to be compiled with the env var `PWA_URL=http://pass.animacraft.fr` (if that's some information you can pass along to whoever installed Padloc for you). Also, please consider using SSL/HTTPS instead of unencrypted HTTP.

**Comment by BrunoBernardino** [comment, marked as answer]
Closing since there's been no update for over a month.
